ironfish/src/rpc/routes/wallet/createTransaction.ts

```diff
@@ -218,6 +246,28 @@ routes.register<typeof CreateTransactionRequestSchema, CreateTransactionResponse
       }
     }
 
+    if (request.data.evm) {
+      // if 0x is present, remove it
+      const evm = request.data.evm
+      const to = evm.to.startsWith('0x') ? evm.to.slice(2) : evm.to
```

Comment:
I feel like we're going to be doing this so much that it might be worth adding a util 😅

Reply:
Lol probably true

Reply:
Added